standards.md

```diff
@@ -57,36 +43,72 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
   }
   ```
 
-17. There MUST NOT be any assignment or modification of variables in the expressions of `if` statements.
+14. There MUST NOT be any assignment or modification of variables in the expressions of `if` statements.
 
   ```php
   if ($nutritionPlan = NutritionPlan::get(NutritionPlan::ID_NIBBLE)) {
     // not allowed
   }
   ```
-18. Abstract class names MUST begin with `Abstract`.
+15. Abstract class names MUST begin with `Abstract`.
 
   ```php
   abstract class AbstractGenerator {
   }
   ```
-19. Interface names MUST end with `Interface`.
+16. Interface names MUST end with `Interface`.
 
   ```php
   interface GeneratorInterface {
   }
   ```
-20. A DAL entity SHOULD NOT be at the top-level of Graze\Lib\Entity namespace, for example:
+17. A DAL entity SHOULD NOT be at the top-level of Graze\Lib\Entity namespace, for example:
   - `Graze\Lib\Entity\Account\Account` instead of `Graze\Lib\Entity\Account`
 
-21. DAL repository classes MUST match the namespacing pattern of their respective entity, examples:
+18. DAL repository classes MUST match the namespacing pattern of their respective entity, examples:
   - `Graze\Lib\Entity\Account\Account` -> `Graze\Lib\Repository\Account\AccountRepository`
   - `Graze\Lib\Entity\Account\Profile` -> `Graze\Lib\Repository\Account\ProfileRepository`
   - `Graze\Lib\Entity\Account\StatusMap` -> `Graze\Lib\Repository\Account\StatusMapRepository`
   - `Graze\Lib\Entity\Account\Profile\Group` -> `Graze\Lib\Repository\Account\Profile\GroupRepository`
   - `Graze\Lib\Entity\Account\Profile\GroupMap` -> `Graze\Lib\Repository\Account\Profile\GroupMapRepository`
 
-22. Doc block types for scalar values MUST be one of: `bool` (not boolean), `int` (not integer), `string`, `float` (not double)
+### PHP DocBlock
+1. DocBlock for functions / methods MUST exist where the function / method has arguments or a return value and MUST use the appropriate tags (@param, @return) to denote that. Developers making modifications to a function / method are tasked with ensuring that the DocBlock is up-to-date.
+2. Functions / methods / classes that are sufficiently complex (i.e. not self-documenting) SHOULD have a DocBlock explaining how to use the function / method / class.
+3. DocBlock presenting the type MUST be present for class member variables.
+4. When a description is necessary for class member variables, the DocBlock MUST be multiline:
+
+  ```php
+  class Foo
+  {
+      /** @var int */
+      protected $id;
+      
+      /**
+       * The Bar used to fight the foo
+       * @var Bar
+       */
+      private $bar;
+  }
+  ```
+
+5. DocBlock types for scalar values MUST be one of: `bool` (not boolean), `int` (not integer), `string`, `float` (not double).
+6. DocBlock types for parameters/return values that are arrays of a single type MUST be written as: `Type[]`. If the variable is a multi-dimensional array, it MUST be represented with one set of `[]` per depth (`int[][]` for an array of array of ints)
```

Comment:
I'm not sure anything needs to change just wondering how useful defining something as multidimensional is, I guess it shows that you need to dig deeper...